docs/tutorials/adding-new-project.md

```diff
@@ -0,0 +1,160 @@
+sure that you are signed in to your GitHub account, and then click the "fork"
+button in the upper-right corner.
+
+<img src="https://github-images.s3.amazonaws.com/help/repository/fork_button.jpg" style="max-width: 300px;" />
+
+It should take a second, but when GitHub finishes processing your request you
+should be at a page very similar to where you started, except now you are
+looking at your own copy.
+
+##Step 2: Setting Up A Preview (Optional)
+
+If you want to be able to preview your new page before you submit it, and are
+willing to add a bit of extra complexity, you have to create a new branch
+called `gh-pages`. Adding this branch will let you preview the whole site, so
+you can see how your page will look when it is published (which is different
+from the markdown preview that you will see on GitHub). To do this, click the
+box where it says "branch: master". It should expand to show you a text box.
+Type the text `gh-pages` in to the box that appears (make sure that it is
+exactly that text; this is important). It should give you the option to create
+a new branch; click that button.
+
+<img src="/images/Website/Branching-on-GitHub.png" style="max-width: 50%;" />
+
+Now, you should be able to navigate to
+`http://<username>.github.io/ev3dev.github.io` in a new tab to see your copy of
+the site. After you complete step 3, visit this preview URL again to make sure
+that your page looks as it should.
+
+If you complete this step, note that everywhere it says "`master`" in the
+screenshots below, it should say "`gh-pages`" instead.
+
+*WARNING*: If you complete this step, GitHub will send you emails warning you
+about the CNAME. You can ignore them; but if they get annoying, see [the readme]
+on the official site repo for information on circumventing the problem.
+
+##Step 3: Adding Your Page
+
+Now that you have your own copy the site, it's time to create your page. To
+start, use the file list and choose the `projects/_posts` folder. Select the
+file called `2014-03-21-Example-Project.md` to open it in the browser.
+
+<img src="/images/Website/Project-template-file-on-GitHub.png" style="max-width: 100%;" />
+
+GitHub will try to read the file and render it how it thinks it's intended to
+be displayed. Although the preview that it shows you probably won't look
+exactly like it will on the site, it should give you a pretty good idea.
+
+Click the "Raw" button in the upper-right corner of the page to view the
+contents of the file itself.
+
+This file that you have opened is not an actual project. It is a template file
+that is meant to be copied to make new project creation easier. The part we
+want is everything between the "---"s. This is called "frontmatter," and it
+describes everything about a project, such as the author of the project. Select
+and copy the frontmatter out of the template file (make sure that you include
+the dashes). This is what you should have:
+
+    ---
```

Comment:
Idea: we can embed GitHub Gists on the website. What if we host the sample page as a Gist so that the content below is always up-to-date and can be copied from here instead of having to view the raw file somewhere else.

https://gist.github.com/benbalter/5555251